deployments/container/reconfigure-mig.sh

```diff
@@ -525,28 +541,31 @@ if [ "${?}" != "0" ]; then
 fi
 
 if [ "${CDI_ENABLED}" = "true" ]; then
-	echo "Running nvidia-smi"
-	chroot ${DRIVER_ROOT_CTR_PATH} nvidia-smi >/dev/null
-	if [ "${?}" != "0" ]; then
-		exit_failed
-	fi
-
 	echo "Creating NVIDIA control device nodes"
-	nvidia-ctk system create-device-nodes --control-devices --driver-root=${DRIVER_ROOT_CTR_PATH}
+	nvidia-ctk system create-device-nodes \
+	  --load-kernel-modules \
+	  --control-devices \
+	  --driver-root=${DRIVER_ROOT_CTR_PATH} \
+	  --dev-root=${DEV_ROOT_CTR_PATH}
```

Comment:
This assumes we add a `--dev-root` option to the `nvidia-ctk system create-device-nodes` command.

Reply:
I have reverted this change for now, and instead have resorted to setting `--driver-root=${DEV_ROOT_CTR_PATH}` since `nvidia-ctk system create-device-nodes` currently creates device nodes there.